constants.py

```diff
@@ -14,72 +25,25 @@
-        "aarch64-debian-11",
-        "aarch64-debian-11-deb-autobake",
-        "aarch64-debian-sid",
-        "aarch64-debian-sid-deb-autobake",
-        "aarch64-fedora-35",
-        "aarch64-fedora-35-rpm-autobake",
-        "aarch64-fedora-36",
-        "aarch64-fedora-36-rpm-autobake",
-        "aarch64-rhel-8",
-        "aarch64-rhel-8-rpm-autobake",
-        "aarch64-rhel-9",
-        "aarch64-rhel-9-rpm-autobake",
-        "aarch64-ubuntu-1804",
-        "aarch64-ubuntu-1804-deb-autobake",
-        "aarch64-ubuntu-2004",
-        "aarch64-ubuntu-2004-deb-autobake",
-        "amd64-centos-stream8-rpm-autobake",
-        "amd64-debian-sid",
-        "amd64-debian-sid-deb-autobake",
-        "amd64-ubuntu-2004",
-        "amd64-ubuntu-2004-deb-autobake",
-        "ppc64le-centos-stream8-rpm-autobake",
-        "ppc64le-debian-11",
-        "ppc64le-debian-11-deb-autobake",
-        "ppc64le-debian-sid",
-        "ppc64le-debian-sid-deb-autobake",
-        "s390x-ubuntu-2004",
-        "s390x-ubuntu-2004-deb-autobake",
-        "s390x-ubuntu-2204",
-        "s390x-ubuntu-2204-deb-autobake",
-        "s390x-rhel-8",
-        "s390x-rhel-8-rpm-autobake",
-        "s390x-rhel-9",
-        "s390x-rhel-9-rpm-autobake",
-        "s390x-sles-15",
-        "s390x-sles-15-rpm-autobake",
+# Special builders triggering
+builders_big = ["amd64-ubuntu-1804-bigtest"]
+builders_dockerlibrary = ["amd64-rhel8-dockerlibrary"]
+builders_eco = [
+        "amd64-debian-10-eco-mysqljs",
+        "amd64-debian-10-eco-pymysql",
+        "amd64-ubuntu-2004-eco-dbdeployer",
+        "amd64-ubuntu-2004-eco-php",
         ]
 
-builders_big=["amd64-ubuntu-1804-bigtest"]
-
-# Generate install builders based on the os_info data
-builders_install = []
-builders_upgrade = []
-builders_autobake = []
-all_platforms = set()
-for os in os_info:
-    for arch in os_info[os]['arch']:
-        all_platforms.add(arch)
-        builder_name_autobake = arch + '-' + os + '-' + os_info[os]['type'] + '-autobake'
-        builders_autobake.append(builder_name_autobake)
-        # Currently there are no VMs for x86 and s390x and OpenSUSE and SLES
-        # RHEL-9 is unreleased, so until packages are available skip install/upgrade
-        if arch not in ['s390x', 'x86'] and os not in ['opensuse-15', 'rhel-9', 'sles-12', 'sles-15']:
-            builders_install.append(builder_name_autobake + '-install')
-            builders_upgrade.append(builder_name_autobake + '-minor-upgrade')
-            builders_upgrade.append(builder_name_autobake + '-major-upgrade')
-
-builders_galera = list(map(lambda x: "gal-" + "-".join(x.split('-')[:3]), builders_autobake))
-
-builders_eco=["amd64-ubuntu-2004-eco-php", "amd64-debian-10-eco-pymysql", "amd64-debian-10-eco-mysqljs", "amd64-ubuntu-2004-eco-dbdeployer"]
+# Defines branches for which we save packages
+savedPackageBranches = branches_main + [
+        "bb-*-release",
+        "bb-10.2-compatibility",
+        "preview-*",
```

Comment:
@vladbogo can we add something like "*pkgtest*" to this list so we can make packaging changes, like [fakeroot removal](https://mariadb.zulipchat.com/#narrow/stream/118759-general/topic/Debian/near/292604558), and examine the resulting packages.

FYI @fauust

Reply:
@grooverdan what do you mean by `"*pkgtest*"`? Something like lintian (https://github.com/MariaDB/buildbot/pull/25)?

Reply:
Nope, adding to the `savedPackageBranches` list.

Reply:
sure